### DISCUSSION TITLE
SharedTransition between stacks

### DISCUSSION THREAD

**Discussion by NischalShahi** [software-mansion/react-native-reanimated#5966] [Q&A] [answered]
I have 2 different stacks with a common component. I want the the component to animate using sharedTransitionTag. I did add the sharedTransitionTag in one screen of stackA and navigated to StackB, the animation does not happen. Is there a way to animate it?

**Comment by tomekzaw** [comment, marked as answer]
Hi @NischalShahi, this is already implemented by @piaskowyk in https://github.com/software-mansion/react-native-reanimated/pull/5793 and will be released in 3.13.0 soon!
